Gemfile.lock

```diff
@@ -267,7 +267,7 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    retriable (3.1.1)
+    retriable (3.1.2)
```

Comment:
I revert these extra version bumps personally.  Save them for a rails upgrade

Reply:
These are dependency upgrades for the SJ common gem